ESPConnector/pom.xml

```diff
@@ -3,11 +3,11 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>inc.softserve</groupId>
-  <artifactId>SendEmailService</artifactId>
+  <artifactId>ESPConnector</artifactId>
   <version>1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>SendEmailService</name>
+  <name>ESPConnector</name>
   <url>http://maven.apache.org</url>
 
   <properties>

```

Comment:
In the properties, not common modules can utilize `${project.version}` variable. Like for `email-core` we may use project version.